qiskit/circuit/library/standard_gates/rx.py

```diff
@@ -90,15 +92,23 @@ def control(
             ctrl_state: control state expressed as integer,
                 string (e.g.``'110'``), or ``None``. If ``None``, use all 1s.
             annotated: indicates whether the controlled gate can be implemented
-                as an annotated gate.
+                as an annotated gate. If ``None``, this is set to ``False``, unless the
+                gate contains free parameters and has more than one control qubit,
+                in which case it cannot yet be synthesized.
```

Comment:
Is this trying to say that `None` will be treated as `True` if the gate contains free parameters and has more than one control? (I believe this is the case). Or that users are required to set it explicitly to `True`? If it's the first one, I would say:

```suggestion
                as an annotated gate. If ``None``, this is set to ``False`` except for 
                the case where the gate contains free parameters and has more than 
                one control qubit, where it is set to ``True``.
```

If it's the second one, I would say:

```suggestion
                as an annotated gate. If ``None``, this is set to ``False``. If the
                gate contains free parameters and has more than one control qubit,
                it cannot yet be synthesized unless this input is set to ``True``.
```

Reply:
I used the RXX message as you suggested below 🙂